components/admin_console/users_and_teams_settings/users_and_teams_settings.jsx

```diff
@@ -88,6 +88,7 @@ export default class UsersAndTeamsSettings extends AdminSettings {
 
     getConfigFromState = (config) => {
         config.TeamSettings.EnableUserCreation = this.state.enableUserCreation;
+        config.TeamSettings.EnableTeamCreation = this.state.enableTeamCreation;
```

Comment:
Not sure if this is the correct way to do this now that the role based permission took effect already.